src/llmcompressor/pipelines/basic/pipeline.py

```diff
@@ -46,9 +46,7 @@ def __call__(
 
         with contextlib.ExitStack() as stack:
             stack.enter_context(calibration_forward_context(model))
-
-            if dataset_args is not None and dataset_args.calibrate_moe_context:
-                moe_calibration_context(model, stack)
+            stack.enter_context(moe_calibration_context(model))
```

Comment:
I dont think we want to do this for every case as not every model will be an MoE